client/src/pages/PageNotFound.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import ArrowForwardIcon from "@material-ui/icons/ArrowForward";
+
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "../assets/happy-cal.png";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Logo from "../assets/logo.png";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginTop: '4rem',
+    flexGrow: 1,
+  },
+  image: {
+    height: "auto",
+    maxWidth: "100%",
+  },
+  imageBox: {
+    maxWidth: "40rem",
+  },
+  title: {
+    color: "orange",
+    fontWeight: "bold",
+  },
+  link: {
+    fontSize: "1.4rem",
+  },
+  nav: {
+    flexGrow: 1,
+    background: "white",
+    padding: "1.5rem 3rem",
+  },
+  logo: {
+    flexGrow: 1,
+  },
+}));
+
+export default function PageNotFound() {
+  const classes = useStyles();
+
+  return (
+    <>
+      <AppBar className={classes.nav} position="static" elevation={0}>
+        <Toolbar>
+          <span className={classes.logo}>
+            <img width="110rem" src={Logo} alt="CalendApp logo" />
+          </span>
+        </Toolbar>
+      </AppBar>
+      <Grid
+        className={classes.root}
+        container
+        justify="center"
+        alignItems="center"
+        spacing={0}
+      >
+        <Grid item className={classes.imageBox}>
```

Comment:
```suggestion
        <Grid sm={12} md={4} item className={classes.imageBox}>
```
Using grid sizing like this will make the page display properly on medium sized screens (like mine)